charts/tekton-pipelines/values.yaml

```diff
@@ -5,8 +5,10 @@
 # -- default images used in our solution
 # @default -- See below
 images:
-  # -- argocd cli image (used in argocd-deploy task)
+  # -- argocd image (used in argocd-deploy task)
```

Comment:
I think `images.argocd` parameter is redundant now and we can delete it to not confuse

Reply:
ok